src/main/java/mate/controller/manufacturer/AddingNewManufacturer.java

```diff
@@ -0,0 +1,33 @@
+package mate.controller.manufacturer;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Manufacturer;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/manufacturers/add")
+public class AddingNewManufacturer extends HttpServlet {
```

Comment:
```suggestion
public class AddManufacturerController extends HttpServlet {
```

Controller name should end for `Controller` word (fix in all places)